x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/DocsV3Support.java

```diff
@@ -381,20 +425,37 @@ private FunctionDocsSupport(String name, Class<?> testClass) {
             super("functions", name, testClass, () -> AbstractFunctionTestCase.signatures(testClass));
```

Comment:
nit: this could use the new, more general c'tor below.

Reply:
I'd rather not add null parameters in more places. I committed your other suggestion because the null parameter was resolved in code just a few lines below, but in this case the null is processed further away, so I'd rather not have it visible here at all.